data/EmaniAditya.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Aditya Reddy",
+  "displayStatusPublic": true,
+  "type": "personal",
+  "bio": "trying to Escape the Matrix. </3 . 🤫 !",
+  "avatar": "https://github.com/EmaniAditya.png",
```

Comment:
Please remove `avatar`, `type`, & `displayStatsPublic` keys from your `json` as we no longer need them. 

```suggestion
  "bio": "trying to Escape the Matrix. </3 . 🤫 !",
```

Reply:
Made the required changes.
![image](https://github.com/EddieHubCommunity/LinkFree/assets/93025567/248b1ec8-14d2-4834-a4d0-9dbe57f68126)